<launch>
  <node name="ft_ros_interface" pkg="ft_ros_interface" type="ft_ros_interface_node" output="screen">
    <param name="poll_rate" type="int" value="100" />
    <param name="ip_address" type="string" value="172.28.103.156" />
    <param name="udp_port" type="int" value="7779" />
    <param name="buffer_size" type="int" value="10" />
  </node>
</launch>
